### DISCUSSION TITLE
I Trying to Tweet from A Cloudflare Worker and I Can't Becuase Cloudflare Workers Don't Support Nodejs's `http`

### DISCUSSION THREAD

**Discussion by ChristianOConnor** [plhery/node-twitter-api-v2#268] [Q&A]
As the title says, I want to periodically tweet from a Cloudflare worker using 'twitter-api-v2.'  The problem is that I can't use 'twitter-api-v2' in a Cloudflare worker because it relies on nodejs's `http`. So I need to get the raw API call that 'twitter-api-v2' is making. This is my code that works locally on a node install.  
index.ts
```
const rwClient = require('./twit.js')

const tweet = async () => {
  try {
    const rawpost = rwClient.v1.tweet("Tweeting from remote third")
    console.log(rawpost)
    const rawpostMade = await rawpost
    console.log(rawpostMade)
  } catch (e) {
    console.error(e)
  }
}

tweet()
```   

twit.ts  
```
const { TwitterApi } = require('twitter-api-v2')

const client = new TwitterApi ({
  appKey: "<APP KEY HERE>",
  appSecret: "APP SECRET HERE",
  accessToken: "<ACCESS TOKEN HERE>",
  accessSecret: "<ACCESS SECRET HERE>",
});

const rwClient = client.readWrite

module.exports = rwClient
```

So what would this look like as a raw request?

**Comment by alkihis** [comment]
Requests are just HTTP call to the right method and the right endpoint mentionned in Twitter documentation. Nothing more, nothing less.
I see that CloudFlare Workers have access to Node.js since last year, so it should normally work?

Otherwise you cannot use this client as it intensively use node's internals.
